File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main()
{
    int height;
    do{
        height = get_int("Height: ");
    }while((height<1) || (height>8));

    int space=height-1;
    int counter=1;

    for(int i=0; i<height; i++)
    {
        for(int j=0; j<space; j++)
        {
            printf(" ");
        }
        
        for(int k=0; k<counter*2; k++)
        {
            printf("#");
            if(k==counter-1)
            {
                printf("  ");
            }
        }
        counter++;
        space--;
        printf("\n");
    }
    return 0;
}

File: sentimental-mario-more/mario.py
while True:
    height = input("Height: ")

    if height.isdigit():
        height = int(height)

        if 1 <= height <= 8:
            break


space = height - 1
counter = 1

for i in range(0, height):

    for j in range(0, space):
        print(" ", end="")

    for k in range(0, (counter * 2)):
        print("#", end="")

        if k == (counter - 1):
            print("  ", end="")

    counter = counter + 1
    space = space - 1
    print()
